src/main/java/org/jahia/modules/jahiaauth/impl/cache/JahiaAuthMapperServiceImpl.java

```diff
@@ -115,13 +115,18 @@ public void executeMapper(String sessionId, MapperConfig mapperConfig, Map<Strin
                 throw new JahiaAuthException("Could not execute mapper: missing mandatory property");
             }
             if (propertiesResult.containsKey(mapping.getConnectorProperty())) {
-                mapperResult.put(mapping.getMappedProperty(), new MappedProperty(m != null ? m.get(mapping.getMappedProperty()) : new MappedPropertyInfo(mapping.getMappedProperty()), propertiesResult.get(mapping.getConnectorProperty())));
+                mapperResult.put(mapping.getMappedProperty(), new MappedProperty(m != null ? m.get(mapping.getMappedProperty()) : new MappedPropertyInfo(mapping.getMappedProperty()),
+                        handlePropertyNullable(propertiesResult.get(mapping.getConnectorProperty()))));
             }
         }
 
         return mapperResult;
     }
 
+    private Object handlePropertyNullable(Object property) {
```

Comment:
you don't need a function to call a function .. 
if you want to improve the code, move the `String.valueOf(propertiesResult.get(mapping.getConnectorProperty())` into a variable with a comment.